src/Common/tests/System/Net/Configuration.Http.cs

```diff
@@ -14,7 +14,7 @@ public static partial class Http
 
             public static string SecureHost => GetValue("COREFX_SECUREHTTPHOST", DefaultAzureServer);
 
-            public static string Http2Host => GetValue("COREFX_HTTP2HOST", "http2.akamai.com");
+            public static string Http2Host => GetValue("COREFX_HTTP2HOST", "www.microsoft.com");
```

Comment:
As you know from the other email thread, that is a bug and not a difference we want to test against. So, it seems better to have only one server listed here. Eventually when the winHTTP bug is fixed, there would be separate native tests for that anyway.